SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/DeploymentLauncher/DeploymentLauncher.csproj

```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>DeploymentLauncher</RootNamespace>
     <AssemblyName>DeploymentLauncher</AssemblyName>
-    <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
```

Comment:
Is this a deliberate change?

Reply:
Yeah, this was required to update the PlatformSDK - we were pinned on an older version (13.7.1)

Reply:
Although review is open to whether we should update this for 0.6.3

Reply:
I don't think we're meant to be changing this but we need to make sure whether everything works in 13.7.1

Reply:
Hmmmm, it's fair that we might not want to do this for 0.6.3-rc specifically. I think we should be updating the PlatformSDK in later versions if we can though.